docs/tutorials/trajectories.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+Handle Expert Trajectories
+==========================
+
+For imitation learning we need trajectories.
+Some are generated by experts and serve as demonstrations,
+others are generated by the agent and serve as training data for a discriminator.
+In this library they are stored in a :class:`Trajectory <imitation.data.types.Trajectory>` dataclass:
+
+.. code-block:: python
+
+    @dataclasses.dataclass(frozen=True)
+    class Trajectory:
+        obs: np.ndarray
+        """Observations, shape (trajectory_len + 1, ) + observation_shape."""
+
+        acts: np.ndarray
+        """Actions, shape (trajectory_len, ) + action_shape."""
+
+        infos: Optional[np.ndarray]
+            """An array of info dicts, shape (trajectory_len, )."""
+
+        terminal: bool
+        """Does this trajectory (fragment) end in a terminal state?"""
+
+:class:`TrajectoryWithRew <imitation.data.types.TrajectoryWithRew>` is a subclass of
+:class:`Trajectory <imitation.data.types.Trajectory>` and has another
+:class:`rews <imitation.data.types.TrajectoryWithRew.rews>` field,
+which is an array of rewards of shape `(trajectory_len, )`.
+
+Usually, they are passed around as sequences of trajectories.
+
+Some algorithms do not care about entire trajectories, but only for individual
+:class:`Transitions <imitation.data.types.Transitions>` (:func:`flattened <imitation.data.rollout.flatten_trajectories>` trajectories).
+
+Generating Trajectories
+-----------------------
+To generate trajectories from a given policy, run the following command:
```

Comment:
Might be good to explain what `sample_until` does here i.e. does it sample complete episodes or does it cut off the last episode.

Reply:
Good point. The kwarg should have been `min_timesteps` instead of `timesteps`. Then it is more clear.

I don't want to get lost in details in this high-level tutorial so decided to just go with `min_episodes`. Then the questions does not even come up when reading the example code.